gmso/core/element.py

```diff
@@ -170,8 +181,8 @@ def element_by_mass(mass, exact=True):
         matched_element = mass_dict.get(mass_trimmed)
     else:
         # Closest match mode
-        mass_closest = min(mass_dict.keys(), key=lambda k: abs(k-mass_trimmed))
-        msg2 = 'Closest mass to {}: {}'.format(mass_trimmed, mass_closest)
+        mass_closest = min(mass_dict.keys(), key=lambda k: abs(k - mass_trimmed))
```

Comment:
Might be worthwhile to have a small note here or elsewhere about min, that it will only return the first minimal item.